datadog_checks_base/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - datadog_checks_base
 
+## 11.2.0 / 2020-03-07
+
+* [Added] Optional `send_histogram_counters_with_distributions` for OpenMetrics. See [#5953](https://github.com/DataDog/integrations-core/pull/5953).
+
```

Comment:
No idea if I did this right and just chose a "reasonable" date in the future to have this released by 🤷‍♂ 

Obviously not expecting this to necessarily be done in a week.